packages/sync/src/modules/Options.php

```diff
@@ -61,20 +131,42 @@ function enqueue_full_sync_actions( $config, $max_items_to_enqueue, $state ) {
 		 */
 		do_action( 'jetpack_full_sync_options', true );
 
-		// The number of actions enqueued, and next module state (true == done)
+		// The number of actions enqueued, and next module state (true == done).
 		return array( 1, true );
 	}
 
-	public function estimate_full_sync_actions( $config ) {
+	/**
+	 * Retrieve an estimated number of actions that will be enqueued.
+	 *
+	 * @access public
+	 *
+	 * @param array $config Full sync configuration for this sync module.
+	 * @return array Number of items yet to be enqueued.
```

Comment:
Sorry, this is not an array, but an integer.
```suggestion
	 * @return int Number of items yet to be enqueued.
```

Reply:
Thanks for fixing it! ❤️